CHANGELOG.rst

```diff
@@ -15,6 +15,8 @@ Next Version
    * Update MOAB, OpenMC, hdf5 versions in apt build (#1529)
    * Add dockerfile and workflow to build PyNE with Conda (#1540)
    * Remove deprecated references to ``basestring`` (#1544)
+   * Add CI for MacOS (#1557)
```

Comment:
I think this line is a carry over from a previous branch point - this line is above in the CHANGELOG